package.json

```diff
@@ -1,49 +1,53 @@
 {
-	"name": "generator-meanjs",
-	"version": "0.1.12",
-	"description": "MEAN.JS Official Yeoman Generator",
-	"main": "app/index.js",
-	"repository": "meanjs/generator-meanjs",
-	"author": {
-		"name": "MEAN.JS Team",
-		"email": "support@meanjs.org",
-		"url": "https://github.com/meanjs"
-	},
-	"engines": {
-		"node": ">=0.10.0"
-	},
-	"scripts": {
-		"test": "mocha"
-	},
-	"keywords": [
-		"yeoman-generator",
-		"scaffold",
-		"framework",
-		"component",
-		"front-end",
-		"app",
-		"mean",
-		"meanjs",
-		"node",
-		"express",
-		"angular"
-	],
-	"dependencies": {
-		"yeoman-generator": "~0.17.1",
-		"chalk": "~0.5.1",
-		"underscore.inflections": "~0.2.1"
-	},
-	"devDependencies": {
-		"assert": "~1.1.1",
-		"async": "~0.9.0",
-		"fs-extra": "~0.10.0",
-		"mocha": "~1.21.3",
-		"temp": "~0.8.0"
-	},
-	"peerDependencies": {
-		"yo": ">=1.0.0"
-	},
-	"licenses": [{
-		"type": "BSD"
-	}]
-}
\ No newline at end of file
+  "name": "generator-meanjs",
+  "version": "0.2.0",
+  "description": "MEAN.JS Official Yeoman Generator",
+  "main": "app/index.js",
+  "repository": "meanjs/generator-meanjs",
+  "author": {
+    "name": "MEAN.JS Team",
+    "email": "support@meanjs.org",
+    "url": "https://github.com/meanjs"
+  },
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "keywords": [
+    "yeoman-generator",
+    "scaffold",
+    "framework",
+    "component",
+    "front-end",
+    "app",
+    "mean",
+    "meanjs",
+    "node",
+    "express",
+    "angular"
+  ],
+  "dependencies": {
+    "bluebird": "^2.9.34",
+    "c.log": "0.0.3",
+    "chalk": "~0.5.1",
+    "underscore.inflections": "~0.2.1",
+    "yeoman-generator": "~0.17.1"
+  },
+  "devDependencies": {
+    "assert": "~1.1.1",
+    "async": "~0.9.0",
+    "fs-extra": "~0.10.0",
+    "mocha": "~1.21.3",
+    "temp": "~0.8.0"
+  },
+  "peerDependencies": {
+    "yo": ">=1.0.0"
+  },
+  "licenses": [
+    {
+      "type": "BSD"
```

Comment:
@ilanbiala I agree that it should me MIT. It was the license that was in the master branch.